Woof/Woof/View/SitterFlow/SitterWalkingsView/SitterWalkingsView.swift

```diff
@@ -4,11 +4,56 @@ import SwiftUI
 struct SitterWalkingsView: View {
     var body: some View {
         NavigationView {
-            Text("Walkings stub")
-                .navigationTitle("Walkings")
-                .navigationBarTitleDisplayMode(.inline)
+            ScrollView {
+                SitterWalkingsSectionView(
+                    isExpanded: $showAllFutureWalkings,
+                    walkings: viewModel.futureWalkings,
+                    headerTitle: futureWalkingSectionTitle
+                )
+
+                SitterWalkingsSectionView(
+                    isExpanded: $showCurrentWalkings,
+                    walkings: viewModel.currentWalkings,
+                    headerTitle: currentWalkingSectionTitle
+                )
```

Comment:
I think we should swap the current and further walkings, because we have the only one current walking and there can be a lot of further ones:
```suggestion
   SitterWalkingsSectionView(
                    isExpanded: $showCurrentWalkings,
                    walkings: viewModel.currentWalkings,
                    headerTitle: currentWalkingSectionTitle
                )

                SitterWalkingsSectionView(
                    isExpanded: $showAllFutureWalkings,
                    walkings: viewModel.futureWalkings,
                    headerTitle: futureWalkingSectionTitle
                )

```

Reply:
Rearranged the views, I had the same idea, as we rarely will have more the one current walking, but the amount of future walking could be enormous.